test/stdlib/Accelerate.swift

```diff
@@ -217,7 +217,7 @@ if #available(iOS 9999, macOS 9999, tvOS 9999, watchOS 9999, *) {
         vectorB = [16]
         vectorC = [5]
         
-        vDSP.multiply(vectorSum: (vectorA, vectorB),
+        vDSP.multiply(vectorAddition: (vectorA, vectorB),
```

Comment:
I don't think the `vector` prefix is buying you any clarity. I might write this something like:
`multiply(a, sum: (b, c), result: d)`.

But, I wonder if it wouldn't be better to hold these fused operations out for this year and see if we can't come up with some way to create them at compile-time from the single-operation functions instead. That would be significantly clearer in the long run.

Reply:
I agree on removing the `vector`, `scalar`, and other prefixes. I think there's value in keeping these fused operations - I think it might seem a little arbitrary to omit some functions from this family.